app/controllers/simplified-account/settings/organisation-details/organisation-details-validation.js

```diff
@@ -0,0 +1,9 @@
+const { body } = require('express-validator')
+
+const ORGANISATION_NAME_MAX_LENGTH = 100
+
+module.exports.validators = [
+  body('organisation-name').trim()
+    .notEmpty().withMessage('Enter an organisation name')
+    .isLength({ max: ORGANISATION_NAME_MAX_LENGTH }).withMessage(`Organisation name  must be ${ORGANISATION_NAME_MAX_LENGTH} characters or fewer`)
```

Comment:
double space after Organisation name

Reply:
Good catch

Reply:
Oh this file was actually from my first attempt at writing this and I should have deleted it. oops